src/styles/style.css

```diff
@@ -137,15 +137,17 @@ h2 {
 
 h3 {
   font-size: var(--gatsbyFontSize-25px);
-  padding-bottom: var(--gatsbySpacing-4);
+  margin-bottom: var(--gatsbySpacing-4);
   line-height: 2.1875rem;
   margin-top: 2rem;
-  margin-bottom: 0rem;
 }
 
 h4 {
-  font-size: 1.5625rem;
+  font-size: 1.3125rem;
```

Comment:
Lets stick with the same format of variables for sizes here as well. `--gatsbyFontSize-25px` for example

Reply:
Updated